sweagent/agent/models.py

```diff
@@ -429,8 +430,7 @@ class OllamaModel(BaseModel):
 
     def __init__(self, args: ModelArguments, commands: list[Command]):
         super().__init__(args, commands)
-        from ollama import Client
-        self.client = Client(host=args.host_url)
+        self.client = OllamaClient(host=args.host_url)
```

Comment:
As I mentioned in the comment below, I changed it to a format that can be imported from tests like `patch("sweagent.agent.models.OllamaClient")`.